pwa-devdocs/src/tutorials/intercept-a-target/add-new-environment-variable/index.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Add a new environment variable
+---
+
+Environment variables provide values that may vary across different instances of the same project.
+For example, the `MAGENTO_BACKEND_URL` environment variable tells your storefront project which Magento instance it uses during runtime.
+The value of this variable can vary between development, staging, and production environments.
+
+In PWA Studio storefront projects, the `.env` file in the project's root directory lists the environment variables and their values.
+These variables are available to your Node scripts, but for security reasons, PWA Studio limits which variables your frontend code can access.
+
+This tutorial teaches you how to create a package that provides a React component that uses an environment variable.
+
+## Tasks overview
+
+1. Initialize the project
+2. Create the PlaceholderImage component
+3. Create and register the intercept file
+4. Define the intercept file
+5. Test on a local instance
+
+## Initialize the project
+
+Use [`npm init`][] or [`yarn init`][] to create a new JavaScript package project for this tutorial.
+
+{: .bs-callout .bs-callout-info}
+**Note:** This is a standalone project.
+You do not need to create this inside a storefront project.
+
+Edit the `package.json` file so it looks like the following:
+
+```json
+{
+  "name": "PlaceholderImage",
+  "version": "1.0.0",
+  "license": "MIT",
+  "peerDependencies": {
+    "@magento/pwa-buildpack": "^7.0.0",
+    "@magento/venia-ui": "^5.0.0",
+    "react": "^16.14.0"
+  }
+}
+```
+
+## Create the PlaceholderImage component
+
+Create a React component that renders an image from an image placeholder service source provided by an environment variable.
+
+```sh
+mkdir -p src/components/PlaceholderImage && touch src/components/placeholderImage.js
+```
+
+Inside the `placeholderImage.js`, add the following content:
+
+```jsx
+import React from "react";
+
+const PlaceholderImage = (props) => {
+  const { width = 300, height = 300 } = props;
+
+  const urlTemplate = process.env.IMAGE_PLACEHOLDER_SERVICE_URL;
+
+  if (!urlTemplate) {
+    console.error("Image placeholder service not defined!");
+    return undefined;
```

Comment:
React prefers `null` if you're not going to render anything, causes this error: `Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.`
```suggestion
    return null;
```